app/src/main/java/org/fossasia/pslab/activity/PerformExperimentActivity.java

```diff
@@ -12,28 +12,34 @@
 import org.fossasia.pslab.R;
 import org.fossasia.pslab.adapters.PerformExperimentAdapter;
 
+import butterknife.BindView;
+
```

Comment:
You missed the `butterknife.Butterknife` import here and also the bind command. As a result, when an experiment is opened in the apk you provided, the app crashes.